apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -336,52 +336,79 @@ export class DetailViewContents extends React.Component {
     }
   };
 
-  renderTopSection = () => {
-    const fitWorkshopLink = (
-      <a href={this.props.applicationData.fit_workshop_url} target="_blank">see workshop</a>
-    );
-    const pdWorkshopLink = (
-      <a href={this.props.applicationData.pd_workshop_url} target="_blank">see workshop</a>
+  renderTopSection = () => (
```

Comment:
do you need an explicit `return` statement?

Reply:
No, because it's an expression. If you'd prefer I can rewrite it as a function again:
```jsx
renderTopSection = () => {
  return (
    // ...
  );
}
```